mozilla-release/mobile/android/base/strings.xml.in

```diff
@@ -656,7 +656,7 @@
   <string name="pref_what_is_human_web">&pref_what_is_human_web;</string>
   <string name="pref_enable_human_web">&pref_enable_human_web;</string>
   <string name="pref_category_human_web_summary">&pref_category_human_web_summary;</string>
-  <string name="pref_human_web_url">https://cliqz.com/&formatS1;/whycliqz/human-web</string>
+  <string name="pref_human_web_url">https://cliqz.com/en/whycliqz/human-web</string>
```

Comment:
@luciancor what do you suggest here? I spoke with Christian and he told me the language should be automagically detected on the server side (but it doesn't work, at least to me)